cluster/addons/any-volume-datasource/crd/hello-populator-crd.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: hellos.hello.k8s.io
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+  annotations:
+    api-approved.kubernetes.io: unapproved
```

Comment:
If this is a test, it should use example.com

Reply:
Fixed as suggested.